Mathlib/Data/List/Defs.lean

```diff
@@ -436,7 +436,7 @@ Example: suppose `l = [1, 2, 3]`. `mapDiagM' f l` will evaluate, in this order,
 `f 1 1`, `f 1 2`, `f 1 3`, `f 2 2`, `f 2 3`, `f 3 3`.
 -/
 def mapDiagM' {m} [Monad m] {α} (f : α → α → m Unit) : List α → m Unit
--- as ported:
+    -- as ported:
```

Comment:
```suggestion
-- as ported:
```

Reply:
unindented as suggested and moved the comment below.